src/style.css

```diff
@@ -0,0 +1,31 @@
+.body {
```

Comment:
usually we don't use class selector for `body` and `html` tag. we use tag selectors directly